db/migrate/20200312221631_add_opt_out_rem_me_to_service_providers.rb

```diff
@@ -0,0 +1,5 @@
+class AddOptOutRemMeToServiceProviders < ActiveRecord::Migration[5.1]
+  def change
+    add_column :service_providers, :opt_out_rem_me, :boolean, default: false, null: false
```

Comment:
I don't think we need a new column for this. We already have a column named `aal` which we already use for changes around remember me. I think we should be able to use that

Reply:
Here is where we use the AAL level of the SP to determine how long remember device is good for: https://github.com/18F/identity-idp/blob/master/app/decorators/service_provider_session_decorator.rb#L124-L130

I think we should do the same for whether the checkbox is checked by default

Reply:
Hoops I thought about that but then it's coupled with AAL.

Reply:
I could set it to default to not checked if the aal level is set to 2 also

Reply:
It's supposed to be coupled to AAL. The SPs that want to opt out of this are the ones that want strict AAL2

Reply:
I thought there was going to be an aal2 and a "strict" aal2 for this?

Reply:
Agree that we can bind the functionality to when the AAL level set to 2 for this story (i.e. we don't need an AAL1 SP to be able to have the box unchecked at this time).

Reply:
ok binding it to `aal=2`